windows/Clipboard/Clipboard.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include "pch.h"
+
+#include <functional>
+#include <string>
+#include <NativeModules.h>
+#include <../include/ReactWindowsCore/cdebug.h>
+#include <../Common/unicode.h>
+
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Windows.ApplicationModel.DataTransfer.h>
+
+using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Foundation;
+namespace datatransfer = winrt::Windows::ApplicationModel::DataTransfer;
+
+namespace NativeClipboard {
+	REACT_MODULE(ClipboardModule, L"RNCClipboard");
+	struct ClipboardModule
+	{
+
+		REACT_METHOD(GetString, L"getString");
+		void GetString(bool error, React::ReactPromise<React::JSValue>&& result) noexcept
+		{
+			if (error) {
+				result.Reject("Failure");
+			}
+			else {
+				auto dataPackageView = datatransfer::Clipboard::GetContent();
+				if (dataPackageView.Contains(datatransfer::StandardDataFormats::Text())) {
+					dataPackageView.GetTextAsync().Completed([result, dataPackageView](IAsyncOperation<winrt::hstring> info, AsyncStatus status) {

```

Comment:
Can you move this out to a method to avoid registering the handler every time this gets called?

Reply:
this only gets called once, it's not a subscription for "clipboard content changed" but for "the operation of getting the content once has completed"